site/themes/carvel/layouts/partials/_header.html

```diff
@@ -36,18 +37,19 @@
 				</li>
 				<li><a href="/community/" {{ if (eq .RelPermalink "/community/") }}class="active"{{ end }}>Community</a></li>
 				<li><a href="/blog/" {{ if or (eq .Page.Section "blog") (eq .Page.Section "tags") }}class="active"{{ end }}>Blog</a></li>
+				<li><a href="/resources/" {{ if (eq .RelPermalink "/resources/") }}class="active"{{ end }}>Community</a></li>
 			</ul>
 			<div class="social">
 				<a href="{{ .Site.Params.links.github_url }}"><img src="/img/github.svg" />GitHub</a>
 				<a href="{{ .Site.Params.links.slack_url }}"><img src="/img/slack.png" />#carvel in Kubernetes Slack</a>
 			</div>
 		</div>
 	</div>
-</header>
-
-{{ if $inSubproject }}
-{{ $info := (index .Site.Params .Section) }}
-<header class="subproject-specific">
+ </header>
+  
+ {{ if $inSubproject }}
+ {{ $info := (index .Site.Params .Section) }}
+ <header class="subproject-specific">
```

Comment:
Again, not sure if these spaces are intended (does this fix a previous spacing error?) or accidental.